### PR TITLE
Update docs for DeepSparse Enterprise

### PR DESCRIPTION
**Pull request by markurtz**

### CODE REVIEW

File: src/content/get-started/install/deepsparse-ent.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+githubURL: "https://github.com/neuralmagic/docs/blob/main/src/content/install/deepsparse.mdx"
+index: 2000
+---
+
+# DeepSparse Enterprise Edition Installation
+
+The [DeepSparse Engine](/products/deepsparse-ent) enables GPU-class performance on CPUs, leveraging sparsity within models to reduce FLOPs and the unique cache hierarchy on CPUs to reduce memory movement.
+The engine accepts models in the open-source [ONNX format](https://onnx.ai/), which are easily created from PyTorch and TensorFlow models.
+
+Currently, DeepSparse is tested on Python 3.7-3.9, ONNX 1.5.0-1.10.1, ONNX opset version 11+ and is [manylinux compliant](https://peps.python.org/pep-0513/).
```
Comment (mgoin) [inline src/content/get-started/install/deepsparse-ent.mdx:14]: Python version goes up to 3.10

File: src/content/get-started/install/deepsparse-ent.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+pip install deepsparse-ent[server]
+```
+
+## YOLO Install
+
+The [Ultralytics YOLOv5](/use-cases/object-detection/deploying) models require extra dependencies for deployment.
+To use YOLO models, install with the following extra option:
+
+```bash
+pip install deepsparse-ent[yolo]         # just yolo requirements
```
Comment (mgoin) [inline src/content/get-started/install/deepsparse-ent.mdx:44]: Doesn't yolo auto install on usage?

Comment (markurtz) [reply src/content/get-started/install/deepsparse-ent.mdx:44]: There are dependencies installed with the yolo flow. For 1.3, we'll be fully removing all auto-install functionality to make this consistent and expected from the user side.

File: src/content/get-started/install/deepsparse-ent.mdx (hunk 1 above)
Comment (markurtz) [reply src/content/get-started/install/deepsparse-ent.mdx]: fixed

File: src/content/get-started/install/deepsparse.mdx [hunk 3]
```diff
@@ -6,14 +6,18 @@ githubURL: "https://github.com/neuralmagic/docs/blob/main/src/content/install/de
...
+Currently, DeepSparse is tested on Python 3.7-3.10, ONNX 1.5.0-1.10.1, ONNX opset version 11+ and is [manylinux compliant](https://peps.python.org/pep-0513/).
 It is limited to Linux systems running on x86 CPU architectures.
 
+The DeepSparse Engine is available in two editions:
+1. [**The Community Edition**](/products/deepsparse) is open-source and free for evaluation, research, and non-production use with our [Engine Community License](https://neuralmagic.com/legal/engine-license-agreement/).
+2. [**The Enterprise Edition**](/products/deepsparse-ent) requires a Trial License or [can be fully licensed](https://neuralmagic.com/legal/master-software-license-and-service-agreement/) for production, commercial applications.
+
 ## General Install
 
 Use the following command to install with pip:
```
Comment (jeanniefinks) [inline src/content/get-started/install/deepsparse.mdx:23]:
```suggestion
Use the following command to install the Community Edition with pip:
```

**Review by jeanniefinks — CHANGES_REQUESTED** [review]
Minor edits

**Comment by markurtz** [comment]
@jeanniefinks updated from your edits

**Review by jeanniefinks — COMMENTED** [review]
Under Package Installations, will those four statements get linked somehow through this LinkCard markdown?

**Pull request merged**